### ISSUE TITLE
Discourse - The Aanmelden buttons for Github and Gitlab in the Registreren modal are hard to reach

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#129]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
<a href="https://community.developer.overheid.nl/t/officiele-bekendmakingen-binnenlandse-zaken-en-koninkrijksrelaties/726">Officiële bekendmakingen (Binnenlandse Zaken en Koninkrijksrelaties)</a>

**Probleembeschrijving:**
Probleem: In de "Registreren" modal in de Chrome en Firefox browsers kan de X-knop en de Aanmelden-knoppen voor GitHub en GitLab aan de rechterkant van het modal alleen in focus worden gebracht door terug te tabben wanneer de focus op het e-mailinvoerveld is, aangezien de focus na het laatste focusbare element in het modal, de link 'gebruiksvoorwaarden', naar buiten het modal gaat naar het browservenster zonder het modal te sluiten.

Mogelijke verbeteringen: Reorganiseer de HTML-structuur zodat de natuurlijke tabvolgorde een logisch patroon volgt en zorg ervoor dat alle interactieve elementen in de modal de juiste tabindex-waarden hebben om de correcte volgorde te behouden. Plaats de GitHub/GitLab- en X-knoppen in de DOM na de link "gebruiksvoorwaarden" en vóór elementen buiten de modal. Voeg focus-trapping toe om te voorkomen dat gebruikers per ongeluk uit de modal kunnen tabben.

**Referentiecode:**
```
<div class="create-account-login-buttons">
            <div id="login-buttons" class="ember-view">  <button class="btn btn-social github" aria-label="Aanmelden met GitHub" tabindex="3" type="button">
      <svg class="fa d-icon d-icon-fab-github svg-icon svg-string" xmlns="http://www.w3.org/2000/svg"><use href="#fab-github"></use></svg>
    Aanmelden met GitHub
  </button>
  <button class="btn btn-social oidc" aria-label="Aanmelden met GitLab" tabindex="3" type="button">
      <svg class="fa d-icon d-icon-sign-in-alt svg-icon svg-string" xmlns="http://www.w3.org/2000/svg"><use href="#sign-in-alt"></use></svg>
        Aanmelden met GitLab
  </button>
<!----></div>
          </div>
```

**Gerelateerde WCAG:**
WCAG 2.4.3 Focus order
Level A

**Referentieafbeelding:**

![A widget shows the backwards tabbing focus order of the Registreren modal. The buttons on the right side of the modal can only via reached via back tabbing due to a lack of focus trap on the modal](https://github.com/user-attachments/assets/ba2f5d1e-5809-45b0-b885-fe00c5db8dc5)

_**Referentienummer:** A131_
